bcgov/bc-wallet-mobile
# fix: updated packages for latest proof request changes

## What's changed:
- updated bifold packages to get latest proof request attribute changes

## Testing:
- Get a credential
- Make a proof request against that credential with an attribute that is not available on the credential
- See proof request

![Media (3)](https://github.com/user-attachments/assets/10c4a0ea-bc08-4430-9fb6-c2ca987770fd)

![Media (2)](https://github.com/user-attachments/assets/393754a7-c26a-45fd-9f46-88ab05729e9d)
